src/components/landing/layout/footer/powered-by-links.tsx

```diff
@@ -15,15 +15,15 @@ export const PoweredByLinks: React.FC = () => {
   return (
     <p>
       <Trans i18nKey="landing.footer.poweredBy">
-        <ExternalLink href="https://codimd.org" text="CodiMD"/>
+        <ExternalLink href="https://codimd.org" text="CodiMD" id='codimd'/>
```

Comment:
I don't think this is a good id, because it's too general. Why do you need an id? Can't you do a query for a link that points to this url? Like `a[href=https://CodiMD.org]` . See css attribute selector.

Reply:
fixed that